tests/test/libs/helpers/helpers.bash

```diff
@@ -0,0 +1,23 @@
+function Create-AWS-Creds () {
+  mkdir "${1}/.aws"
+  cat << EOF > "${1}/.aws/credentials"
+[${2}]
+aws_access_key_id=XXXXXXXXXXXXXXXXXX
+aws_secret_access_key=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
Is it ok to keep placeholders like "XXXXXXXXXXXX" ?

Reply:
haven't catch this? please provide details

Reply:
it's just for imitation